### ISSUE TITLE
No tray icon

### ISSUE THREAD

**Issue by vertigo220** [File-New-Project/EarTrumpet#1411]
### Summary

I went to adjust the volume of an app and found the tray icon unresponsive. Since I couldn't do anything, including close it, I killed the process and relaunched it. However, I've done so several times now and every time it's running in tasks but there's no tray icon at all.

### Steps to reproduce

N/A

### EarTrumpet version

Don't know since I can't access it

### Windows version

10.0.19045.2965

**Comment by riverar** [comment]
That's no good, sorry about that. You can also retrieve the version via `Get-AppxPackage *EarTrumpet*`. I presume you installed this via Chocolatey or the Microsoft Store?

**Comment by vertigo220** [comment]
2.2.2.0, and I downloaded the .appinstaller file and used that to install it. Possible I originally installed with Chocolatey, though. But I had lots of issues with that and I think I uninstalled from that and installed normally.

**Comment by vertigo220** [comment]
So I'm not sure what's new, but this started roughly 2-3 weeks ago, and it's happening on a daily basis. I always have a couple dozen or so programs running in the tray, and every day the same two have issues, this and SoundSwitch, both audio device management apps. They've both been running for several months, if not a few years, without issue, until recently. The only thing I can think that might break them is switching in and out of a full-screen (windowed) game, but that's not new. Otherwise, the only thing that's changed that might be related is I switched from an old Nvidia GPU to a new(er) AMD one, and I did that around the same time these issues started, though I'm not certain which was first. But the more I think about it, the more convinced I am that must be the cause somehow.

As far as what's happening, both programs will just be unresponsive when I try to use them (and ALL other programs running in the tray will work fine). The only thing I can do at that point is to force kill them, but then when I try to relaunch them, even though the processes are running, they don't show back up in the tray for a while. Sometimes it takes a couple minutes, sometimes several. And today, I used it to switch audio devices then started playing a game, and twice while playing, quite a while after I interacted with EarTrumpet, it popped up in front of the game, blocking part of it. The first time I clicked back into the game and it went away (or at least was moved below the game so it wasn't visible), but the second time I couldn't get it to close, and I had to kill the task just to get it to stop blocking the game.

And as I've been typing this, I actually relaunched it due to having closed it then, and it's still not showing in the tray.

**Comment by thethomaseffect** [comment]
After a windows update I had the tray icon disappear. I solved the issue by right clicking the taskbar and clicking taskbar settings, then going to system tray icons and clicking Other system tray icons. After some scrolling I seen that EarTrumpet had been switched off somehow by the update. I turned it on again and dragged it to it's previous position and everything is working, but it's possible future Windows updates might have the same effect.
